scripts/standardise_validate.py

```diff
@@ -58,6 +58,13 @@ def parse_args() -> argparse.Namespace:
         required=True,
     )
     parser.add_argument("--cutline", dest="cutline", help="Optional cutline to cut imagery to", required=False, nargs="?")
+    parser.add_argument(
```

Comment:
As we have a control on passing the option `--create-footprint`, why not don't pass it when we don't want to create a capture area? That would remove this complexity below.